sdcm/cluster.py

```diff
@@ -1785,6 +1786,9 @@ def node_run_stress(node, loader_idx, cpu_idx, keyspace_idx, profile, stress_cmd
                                                           cpu_idx, ks_idx, profile, stress_cmd))
                     setup_thread.daemon = True
                     setup_thread.start()
+                    if first_loader:
```

Comment:
We should sleep for creating all keyspaces, not just for first keyspace.

`if loader_idx == 0 and cpu_idx == 0:`

Reply:
You are right.
First, I didn't even think of the multiple keyspaces.
Second, we have the loader_idx and cpu_idx, so I could use it...